docs/Developer's ReadMe.md

```diff
@@ -123,25 +123,31 @@ Set up your Theia workspace for development purposes.
 
 **Follow these steps**:
 
-1. Build a VSIX file with your changes.
+1. Make sure you have the following installed:
 
-2. Build and run the Theia browser example, using [Setting up your Theia workspace](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Theia.md#setting-up-your-theia-workspace).
+   - [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/new/)
+   - [Mozilla Geckodriver](https://github.com/mozilla/geckodriver/releases) (and add it to your PATH)
+   - [ChromeDriver](https://chromedriver.chromium.org/) (and add it to your PATH)
```

Comment:
I saw the current site will be deprecated soon. Perhaps we should point to the new site: https://sites.google.com/chromium.org/driver/ already?

Reply:
Yes, sounds like a good idea 👍 Also I'm adding Google Chrome here because both browsers are necessary, I think.